### ISSUE TITLE
Quest ID 1156 Not Completing (iron tiped bolts)

### ISSUE THREAD

**Issue by shawnbyday** [GTNewHorizons/NewHorizons#3808]
#### Which modpack version are you using?
2.0.5.1
#
#### If in multiplayer; On which server does this happen? 
Private server.
#
#### What did you try to do, and what did you expect to happen?
Made some iron tipped bolts. Expected quest to complete. This happened to me a while back but I'm opening issue because someone else was saying same thing tonight. When I first started playing GTNH it was broken, then it was fixed, now it is broken again.
#
#### What happend instead? (Attach screenshots if needed)
Quest didn't complete.
#
#### What do you suggest instead/what changes do you propose?
Have the quest detect the bolts? :D I'm actually going to look at it myself and do some testing on it with some different ideas and try to fix it if those ideas don't work. Worst case if I find out why I can update the quest description. Can assign to me if you want.

**Comment by Dream-Master** [comment]
will check

**Comment by Dream-Master** [comment]
@shawnbyday did you find out why it wont complete ?

**Comment by shawnbyday** [comment]
@Dream-Master I haven't yet. I just got home about 8 minutes ago and I'm waiting for the game to load so I can do some testing :D I'll post back in a bit.

**Comment by shawnbyday** [comment]
@Dream-Master so I setup the infrastructure to test this and it detected everything except the last part where it detects the iron bolts.

After that I opened the quest book with edit mode enabled and used the quest building interface to select the iron bolts I just made then got out of that and turned edit mode off and it detected the bolts.

After all of that I compared the current DefaultQuest.json file to the one I just did /bq_admin default save on and my merge tool reported that they were identical. So even though the files were the same before and after something I did by editing the file made it detect. It doesn't make sense.

I'm currently loading the game to try again in a fresh world to see if I can recreate.
